### PR TITLE
Create a portfolio

### PR DESCRIPTION
**Pull request by MichaelaTSH**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,159 @@
...
-  </body>
-</html>
+    <!-- .container -->
+
+    <script src="scripts/jquery.min.js"></script>
+    <!-- <script src="js/jquery-3.2.1.slim.min.js"></script> -->
+    <script src="scripts/templatemo-script.js"></script>
+
+</body>
+</html>
```
Comment (kevinluvian) [inline portfolio/src/main/webapp/index.html:159]: usually it is a convention to add a new line at the end of the file

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,159 @@
...
+                </p>
+            </div>
+        </footer>
     </div>
-  </body>
-</html>
+    <!-- .container -->
+
+    <script src="scripts/jquery.min.js"></script>
+    <!-- <script src="js/jquery-3.2.1.slim.min.js"></script> -->
```
Comment (kevinluvian) [inline portfolio/src/main/webapp/index.html:155]: lets remove this if it is not used

File: portfolio/src/main/webapp/scripts/templatemo-script.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+// https://codepen.io/AmruthPillai/pen/axvqB
+(function ($) {
```
Comment (kevinluvian) [inline portfolio/src/main/webapp/scripts/templatemo-script.js:2]: very cool!

File: portfolio/src/main/webapp/unused/script.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        $(this).toggleClass("floating-label-form-group-with-value", !!$(e.target).val());
+      }).on("focus", ".floating-label-form-group", function() {
+        $(this).addClass("floating-label-form-group-with-focus");
+      }).on("blur", ".floating-label-form-group", function() {
+        $(this).removeClass("floating-label-form-group-with-focus");
+      });
+    });
+  
+})(jQuery); // End of use strict
+
```
Comment (kevinluvian) [inline portfolio/src/main/webapp/unused/script.js:98]: seems there's newline problem here

File: portfolio/src/main/webapp/unused/script.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
```
Comment (kevinluvian) [inline portfolio/src/main/webapp/unused/script.js:1]: if the file is unused, it is usually better to just delete it

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (MichaelaTSH) [reply portfolio/src/main/webapp/index.html:159]: I see, thank you so much for the advice! I'll take note of that in the future.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (MichaelaTSH) [reply portfolio/src/main/webapp/index.html:155]: I have removed all the unused files and that line of code. Thank you so much!

**Comment by kevinluvian** [comment]
I think you may have some issue with centering the content with flexbox.

this is what I usually use:
if there is a parent div which use "display: flex", to center the child where the child width is less than the parent, you can just use "margin: auto"

for example:
<div class="parent">
    <div class="child"></div>
</div>

.parent {
    display: flex
}
.child {
    margin: auto
    width: ...px
}

you may want to edit the project-item, tm-timeline-item-inner, paragraph class to make it better centered, instead of using fixed number of padding/margin
fixed number of padding/margin is usually not a good idea, since there are many devices (laptop/pc/mobile/etc) with different screen resolution, so using a fixed number of padding might look good in your screen, but may not be in others especially if the resolution is different

However since this is just a small issue, we can do that in other PR 👍

**Pull request merged**